src/views/Flipper.js

```diff
@@ -8,11 +8,11 @@
  */
 
 define(function(require, exports, module) {
-    var Transform = require('../core/Transform');
-    var Transitionable = require('../transitions/Transitionable');
-    var RenderNode = require('../core/RenderNode');
-    var OptionsManager = require('../core/OptionsManager');
-
+    var Transform = require('famous/core/Transform');
```

Comment:
please leave relative path's.  This is done due to allow path resolving to be done explicitly.